runatlantis.io/.vuepress/theme/components/Home.vue

```diff
@@ -192,11 +192,11 @@
   $MQMobile = 719px
   $MQMobileNarrow = 419px
 
-  @import '~@temp/override.styl' // generated from user config
-
   $homeWidth = 960px
   $buttonColor = #34be68
   $darkBackground = rgb(250, 250, 250)
+  .container
+    padding-top 3.6rem
```

Comment:
custom-layout is disappeared. So add padding-top into Home.vue container for navbar.

<img width="664" alt="image" src="https://user-images.githubusercontent.com/915731/208413878-3a4e6fbc-4e37-4809-946b-ce72cbe09311.png">